portality/static/js/formulaic.js

```diff
@@ -1288,11 +1288,10 @@ var formulaic = {
                 if (val && (val.substring(0,7) === "http://" || val.substring(0,8) === "https://") && val.length > 10) {
                     if (this.link) {
                         this.link.attr("href", val);
-                        this.link.html(val);
                     } else {
                         var classes = edges.css_classes(this.ns, "visit");
                         var id = edges.css_id(this.ns, this.fieldDef.name);
-                        that.after('<p><small><a id="' + id + '" class="' + classes + '" rel="noopener noreferrer" target="_blank" href="' + val + '">' + val + '</a></small></p>');
+                        that.after('<p><small><a id="' + id + '" class="button ' + classes + '" rel="noopener noreferrer" target="_blank" href="' + val + '">Open this link</a></small></p>');
```

Comment:
This looks good, just worth quickly checking with the editorial team that they don't rely on the full url being visible as a clickable link, in case they have workflow around it.

Reply:
Fair, I will ask in the chat.

Reply:
They are fine with how it is currently. will revert this one.